doc/source/deploy-on-yarn.rst

```diff
@@ -43,18 +43,17 @@ The specification for each service consists of necessary files and commands that
             # There should only be one instance of the head node per cluster.
             instances: 1
             resources:
-                # The resources for the head node.
+                # The resources for the worker node.
                 vcores: 1
                 memory: 2048
             files:
                 ...
             script:
                 ...
         ray-worker:
```

Comment:
```suggestion
        ray-worker:
            # Number of ray worker nodes to start initially.
            # This can be scaled using 'skein container scale'.
```